exchanges/okx/okx_wrapper.go

```diff
@@ -3050,3 +3049,8 @@ func (e *Exchange) GetCurrencyTradeURL(ctx context.Context, a asset.Item, cp cur
 		return "", fmt.Errorf("%w %v", asset.ErrNotSupported, a)
 	}
 }
+
+// MessageID returns a universally unique id using UUID V7, with hyphens removed to fit the maximum 32-character field for okx
+func (e *Exchange) MessageID() string {
+	return fmt.Sprintf("%x", uuid.Must(uuid.NewV7()))
```

Comment:
```suggestion
	return strings.Replace(uuid.Must(uuid.NewV7()).String(), "-", "", 4)
```

slightly more performant